spec/javascripts/admin/views/editions/_form.spec.js

```diff
@@ -1,512 +1,442 @@
-
-  $select.find('option:contains(World news story)').prop('selected', true).change()
-  ok($('fieldset.foreign-language').is(':visible'), 'fieldset containing foreign language options is not visible')
-})
-
-test("unselecting 'World news story' hides and resets the locale fields", function () {
-  var $select = $('select#edition_news_article_type_id')
-
-  $select.find('option:contains(World news story)').prop('selected', true).change()
-  ok($('fieldset.foreign-language').is(':visible'), 'fieldset containing foreign language options is not visible')
-
-  $('input#create_foreign_language_only').click()
-  ok($('div.foreign-language-select').is(':visible'), 'div containing locale inputs has become visible')
-
-  $('#edition_primary_locale').val('cy').change()
-  equal($('#edition_primary_locale option:selected').val(), 'cy', 'foreign-language selected')
-
-  $select.find('option:contains(News story)').prop('selected', true).change()
 
-  equal($('#edition_primary_locale option:selected').val(), '', 'locale reset back to English')
-  ok($('fieldset.foreign-language').is(':hidden'), 'fieldset containing foreign language options is not hidden')
-})
-
-module('admin-edition-image-fields: ', {
-  setup: function () {
-    $('#qunit-fixture').append(form)
-    $('#qunit-fixture form').append(imageFieldsFieldSet)
-
-    GOVUK.adminEditionsForm.init({
-      selector: 'form#non-english',
-      right_to_left_locales: ['ar']
+    it('shows the locale input when foreign language only is checked', function () {
+      form.find('#create_foreign_language_only').click()
+      expect(form.find('.foreign-language-select').is(':hidden')).toBeFalse()
     })
-    $('.js-hidden').hide()
-  }
-})
-
-test('use_no_image radio buttons toggle visibility of image_uploader selector', function () {
-  ok($('.js-show-image-uploader').is(':hidden'), 'image uploader hidden by default')
-
-  $('#edition_image_display_option_no_image').click()
-  ok($('.js-show-image-uploader').is(':hidden'), 'image uploader shown when "image_display_option_no_image" selected')
 
-  $('#edition_image_display_option_custom_image').click()
-  ok($('.js-show-image-uploader').is(':visible'), 'image uploader hidden when "image_display_option_custom_image" selected')
-
-  $('#edition_image_display_option_organisation_image').click()
-  ok($('.js-show-image-uploader').is(':hidden'), 'image uploader shown when "image_display_option_organisation_image" selected')
-})
-
-module('admin-edition-form-first-published-at: ', {
-  setup: function () {
-    $('#qunit-fixture').append(form)
-    $('#qunit-fixture form').append(firstPublishedAtFieldset)
-
-    GOVUK.adminEditionsForm.init({
-      selector: 'form#non-english',
-      right_to_left_locales: ['ar']
+    it('hides and resets the locale input when foreign language only is unchecked', function () {
+      form.find('#create_foreign_language_only').click().click()
```

Comment:
duplicate `click()` here?

Reply:
It is intentional, as it's to be clicked on and then clicked off. But I can understand that it is not clear to read, so I've merged this into the test above to try make it clear.